src/zzz_od/gui/view/devtools/intel_manage_interface.py

```
@@ -0,0 +1,1151 @@
+                self.weight_values[option] = float(input_box.text()) if input_box.text() else 0.0
+            except ValueError:
+                self.weight_values[option] = 0.0
+
+        weight_file = Path(get_resource_path('config', 'character_weight.json'))
+        data = {}
+        if weight_file.exists():
+            try:
+                with open(weight_file, 'r', encoding='utf-8') as f:
+                    data = json.load(f)
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)